chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -1052,6 +1052,15 @@ Zotero.Utilities.Internal = {
 		}
 		elem.appendChild(menu);
 		return menu;
+	},
+
+	/**
+	 * Quits zotero, optionally restarting. Mostly useful for stubbing in tests.
```

Comment:
Capitalize "Zotero". Remove "Mostly useful…" — perfectly appropriate to have a central function for quitting/restarting without this being about tests